packages/core-ethereum/crates/core-ethereum-misc/Cargo.toml

```diff
@@ -37,7 +41,10 @@ mockall = "0.11.4"
 hex-literal = "0.4"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+tokio = "1"
```

Comment:
Why? `#[async_std::test]` should be sufficient.

Reply:
no, unfortunately, it isn't `ethers::Anvil` *requires* tokio